components/service/glean/build.gradle

```diff
@@ -72,3 +74,17 @@ apply from: '../../../publish.gradle'
 ext.configurePublish(config.componentsGroupId, archivesBaseName, project.ext.description)
 
 apply plugin: 'mozilla.components.tooling.glean-gradle-plugin'
+
+pluginManager.withPlugin('mozilla.components.tools.glean-gradle-plugin') {
```

Comment:
This will always be available 'cuz you apply the plugin yourself, I see.  Was going to ask what happened if the plugin wasn't around.   I expect a build failure; you agree?

On an unrelated note: baking paths like this into the _source_ that will get shipped in Fenix makes reproducible builds impossible.  Why on earth do you need this _at runtime_?

Reply:
Ok.  I'll look into one of these other options.  FWIW, master currently has this issue as well.